tree-sitter-darklang/grammar.js

```diff
@@ -184,6 +184,7 @@ module.exports = grammar({
         alias($.mp_tuple, $.tuple),
         alias($.mp_enum, $.enum),
         alias($.variable_identifier, $.variable),
+        alias(/_/, $.wildcard),
```

Comment:
should we allow `variable_identifier`to start with `_` instead of handling it this way?

Reply:
Hmm, yeah I think so.

I've been thinking, and there's some value to (future) having a specific wildcard pattern: anything with _:
- may not be accessed
- should not be complained about, by the type-checker, for lack of usage (where we might otherwise warn "hey you captured this variable but didn't use it!")